app/services/solidus_bolt/accounts/add_address_service.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  module Accounts
+    class AddAddressService < SolidusBolt::BaseService
+      attr_reader :order, :address, :access_token
+
+      def initialize(order:, address:, access_token:)
+        @order = order
+        @address = address
+        @access_token = access_token
+        super
+      end
+
+      def call
+        add_address
+      end
+
+      private
+
+      def add_address
+        handle_result(HTTParty.post("#{api_base_url}/#{api_version}/account/addresses", build_options))
+      end
+
+      def address_body
+        country = address.country
+        {
+          street_address1: address.address1,
+          street_address2: address.address2,
+          locality: address.city,
+          region: address.state&.abbr,
+          postal_code: address.zipcode,
+          country_code: country.iso,
+          first_name: Spree::Address::Name.new(address.name).first_name,
+          last_name: Spree::Address::Name.new(address.name).last_name,
+          phone: address.phone,
+          email: order.email
+        }.to_json
+      end
```

Comment:
Would it be better to add this method to a decorator for `Spree::Address` ?

Considering that the returned hash could be useful in some other points in the code for bolt.

Reply:
good point
another PR (https://github.com/nebulab/solidus_bolt/pull/89) does indeed use the same code
I'll check with @DanielePalombo which PR will probably get merged first and add the code there

Reply:
decorator implemented in https://github.com/nebulab/solidus_bolt/pull/89/commits/74699e3678f7ced99677e61f6504ccf0019abcad